packages/manager/modules/bm-server-components/src/general-information/inputs/template.html

```diff
@@ -0,0 +1,247 @@
+<!-- dynamic inputs form depending on the OS template -->
+<h5 data-translate="server_configuration_installation_inputs_title"></h5>
+<div class="oui-message oui-message_info">
+    <span
+        class="oui-message__icon oui-icon oui-icon-info-circle"
+        aria-hidden="true"
+    >
+    </span>
+    <p
+        class="oui-message__body"
+        data-translate="server_configuration_installation_inputs_info"
+        data-translate-values="{ t0: $ctrl.installation.selectDistribution.description }"
+    ></p>
+</div>
+<p>&nbsp;</p>
```

Comment:
```suggestion
```
I don't think is really needed, maybe you can add a class on previus element if you want an espace between the oui-message and following of form